ow_plexil/src/plans/TestPanTilt.plp

```diff
@@ -0,0 +1,66 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Simple plan to test Pan and Tilt functions
+
+#include "plan-interface.h"
+
+TestPanTilt: Sequence
+{
+
+    log_info("Testing Pan, should succeed on all tests.");
+    LibraryCall Pan(Degrees = 65);
+    log_info("Current Pan Degree: ", Lookup(PanDegrees));
+    log_info("Current Tilt Degree: ", Lookup(TiltDegrees));
+
+    LibraryCall Pan(Degrees = 0);
+    log_info("Current Pan Degree: ", Lookup(PanDegrees));
+    log_info("Current Tilt Degree: ", Lookup(TiltDegrees));
+
+    LibraryCall Pan(Degrees = 180);
+    log_info("Current Pan Degree: ", Lookup(PanDegrees));
+    log_info("Current Tilt Degree: ", Lookup(TiltDegrees));
+
+    LibraryCall Pan(Degrees = -180);
+    log_info("Current Pan Degree: ", Lookup(PanDegrees));
+    log_info("Current Tilt Degree: ", Lookup(TiltDegrees));
+
+    LibraryCall Pan(Degrees = 350);
+    log_info("Current Pan Degree: ", Lookup(PanDegrees));
+    log_info("Current Tilt Degree: ", Lookup(TiltDegrees));
+
+    LibraryCall Pan(Degrees = -350);
+    log_info("Current Pan Degree: ", Lookup(PanDegrees));
+    log_info("Current Tilt Degree: ", Lookup(TiltDegrees));
+
+    LibraryCall Pan(Degrees = -360);
+    log_info("Current Pan Degree: ", Lookup(PanDegrees));
+    log_info("Current Tilt Degree: ", Lookup(TiltDegrees));
+
+    LibraryCall Pan(Degrees = -45);
+    log_info("Current Pan Degree: ", Lookup(PanDegrees));
+    log_info("Current Tilt Degree: ", Lookup(TiltDegrees));
+
+    LibraryCall Pan(Degrees = 10);
+    log_info("Current Pan Degree: ", Lookup(PanDegrees));
+    log_info("Current Tilt Degree: ", Lookup(TiltDegrees));
+    log_info("PAN TEST FINISHED");
+
+    
+
+    log_info("Testing Tilt, should move to 45 degrees, back to 0, then attempt 200 before colliding and timing out.");
```

Comment:
I don't think we should crash the antenna ;)  Unlike pan, tilt has a limited range that we should look up, and just test that range.  One problem with ending the test with a crashed antenna is that this plan cannot be run again, because it first pans.  No plan that uses the antenna can be run after this one, which is inconvenient.

Also, if the intention here was to test the timeout, I don't think we need to test that either.

Reply:
Yeah, the original purpose was to test the timeout. I got stuck in simulation land and forgot that it should actually simulate a real lander lol... I will make those changes!